lib/logger/pretty-stdout.js

```diff
@@ -84,20 +84,22 @@ function formatRecord(rec) {
   return util.format('%s: %s%s\n%s', level, msg, meta, details);
 }
 
-function RenovateStream() {
-  this.readable = true;
-  this.writable = true;
-  Stream.call(this);
+class RenovateStream {
+  constructor() {
+    this.readable = true;
+    this.writable = true;
+    Stream.call(this);
+  }
+
+  // istanbul ignore next
+  write(data) {
+    this.emit('data', formatRecord(data));
+    return true;
+  }
 }
 
 util.inherits(RenovateStream, Stream);
```

Comment:
@JamieMagee maybe we should convert this to typescript too and use the extends keyword

```javascript
class RenovateStream extends Stream {
...
}
```

Reply:
Sure, will take a look

Reply:
@ViceIce I committed what I have, but I still have to fix a small bug with the `pretty-stdout` conversion. It's splitting some log messages (but not all) over multiple lines.

```
INFO: 
       "0": "U",
       "1": "s",
       "2": "i",
       "3": "n",
       "4": "g",
       "5": " ",
```

Reply:
I've seen that without your change in the logs. So I think it is a independent bug.

But sure, we should fix that too.

Reply:
Looks like sometimes the `IRecord` is an `array`, then the `remainingKeys` contains `0...n`

https://github.com/renovatebot/renovate/blob/cef862181b2d3234b7fdbd150aef1368083e5867/lib/logger/pretty-stdout.js#L73-L75